src/main/java/com/treasure/hunt/jts/geom/Circle.java

```diff
@@ -28,6 +32,10 @@
      */
     public Circle(Coordinate coordinate, double radius, int numOfPoints, GeometryFactory geometryFactory) {
         super(null, null, geometryFactory);
+
+        this.radius = radius;
```

Comment:
Just interested why u added them?

Reply:
Have a look into CircleHighlighter,
it is needed by Ellipse2D in order to convert this into a Circle Shape.

Reply:
cool